scripts/database-cleaning.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+echo 'Warning: this action will completely remove the database '${HOCS_DB_NAME^^}' and all its contents. It cannot be undone!'
+
+read -p 'Are you sure you want to remove the database (y/n)? ' yes_no
+
+if [[ $yes_no = 'y' || $yes_no = 'Y' ]]
+then
+    export PGPASSWORD=${HOCS_PASSWORD}
```

Comment:
You could do something like
```suggestion
    export PGPASSWORD=${HOCS_PASSWORD}
    export PGHOST=${HOCS_DB_HOSTNAME}
    export PGUSER=${HOCS_USERNAME}
    export PGDATABASE=${HOCS_DB_NAME}
```
which would mean you don't have to specify the `-hUd` flags below: